numpy/lib/function_base.py

```diff
@@ -2604,15 +2609,15 @@ def blackman(M):
     >>> plt.plot(window)
     [<matplotlib.lines.Line2D object at 0x...>]
     >>> plt.title("Blackman window")
-    <matplotlib.text.Text object at 0x...>
+    Text(0.5, 1.0, 'Blackman window')
```

Comment:
This probably depends on the matplotlib version

Reply:
I guess my approach has been to update to the expectation of a more recent version. Quite a few of these in the diff now.